Resources/config/services.php

```diff
@@ -62,12 +63,23 @@
         ]);
 
     $services->set('gesdinet.jwtrefreshtoken.authenticator')
```

Comment:
This service, and the corresponding class, should be deprecated and removed when support for Symfony 5.2 and earlier is dropped.